ui/components/app/permission-page-container/permission-page-container.component.js

```diff
@@ -213,8 +247,9 @@ export default class PermissionPageContainer extends Component {
           <PageContainerFooter
             footerClassName="permission-page-container-footer"
             cancelButtonType="default"
-            onCancel={() => this.goBack()}
-            cancelText={this.context.t('back')}
+            // TODO these shouldn't be back for switchEthereumChain requests
```

Comment:
What does "be back" mean?

Reply:
Oh I just meant that the text (and behavior) shouldn't be "back" for when the request is a `permittedChains` permissions request.

Reply:
removed this comment: https://github.com/MetaMask/metamask-extension/pull/24415/commits/50fb7a4adf57ffb86fd1b128328180501edf50db